docs/book/usage/started.md

```diff
@@ -1,8 +1,277 @@
-# Getting started
+# Getting Started with Walrus
 
-You want to get a taste of Walrus? Here are some sites that are built on and/or use Walrus:
+Walrus is a platform for building efficient and resilient data markets, where data is stored as
+*blobs*. Walrus stores all blobs as an array of bytes with a fixed size, and you can store any
```

Comment:
Important: Not quite sure about the "fixed size". Blobs can have arbitrary sizes, although they may be padded to adhere to alignment requirements.